Gemfile

```diff
@@ -21,6 +21,8 @@ gem "stimulus-rails" # Hotwire's modest JavaScript framework [https://stimulus.h
 gem "tailwindcss-rails" # Use Tailwind CSS [https://github.com/rails/tailwindcss-rails]
 gem "turbo-rails" # Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
 gem "ulid-rails"
+gem "omniauth-github", github: "omniauth/omniauth-github", branch: "master"
```

Comment:
Is there a reason to include `HEAD` from the repo rather than the released gem version?